uikit/views/src/main/java/com/sygic/maps/uikit/views/poidetail/PoiDetailBottomDialogFragment.kt

```diff
@@ -102,21 +105,33 @@ open class PoiDetailBottomDialogFragment : AppCompatDialogFragment() {
         preferencesManager = PreferencesManager(requireContext())
 
         viewModel = ViewModelProviders.of(
-            this,
-            PoiDetailInternalViewModel.ViewModelFactory(
-                arguments?.getParcelable(POI_DETAIL_DATA)!!,
-                preferencesManager
-            )
+            this, PoiDetailInternalViewModel.ViewModelFactory(preferencesManager)
         )[PoiDetailInternalViewModel::class.java].apply {
+            this.expandObservable.observe(
+                this@PoiDetailBottomDialogFragment,
+                Observer<Any> { expandBottomSheet() })
+            this.collapseObservable.observe(
+                this@PoiDetailBottomDialogFragment,
+                Observer<Any> { collapseBottomSheet() })
+            this.webUrlClickObservable.observe(
+                this@PoiDetailBottomDialogFragment,
+                Observer<String> { context?.openUrl(it) })
```

Comment:
rozmyslam ci by tu vynimocne nestalo za to, aby `PoiDetailInternalViewModel` bol `AndroidViewModel`, lebo tieto open veci by na app contexte boli v pohode a usetrilo by to dost observables... hm?

Reply:
jsem za ;)